Brewfile

```diff
@@ -14,3 +14,4 @@ cask "xquartz"
 brew "mupdf"
 brew "poppler"
 brew "imagemagick"
```

Comment:
Should "imagemagick" be removed if vips is the default?

Reply:
ImageMagick functionality is still being tested here: https://github.com/rails/rails/blob/35d8890c92b931cecf323d4bdd7e318f20326926/activestorage/test/analyzer/image_analyzer/image_magick_test.rb, so I don't think we should remove it. (I also tried running this file without ImageMagick binaries `convert` and `identify` and 3 of the tests fail)

Reply:
Ah these changes are for working on Rails itself 👍